### DISCUSSION TITLE
Perform a putObject to send the version on a bucket to act as a flag to warn user a new version is available

### DISCUSSION THREAD

**Discussion by remyleone** [goreleaser/goreleaser#2104] [Q&A] [answered]
Hello,

As part of our release process, we have to send an encoded string (the version) as an object in an S3 bucket. This well-known file is fetched by our tool to warn the user a new version is available.
How would you upload this string version? Using something custom?

Best regards

**Comment by caarlos0** [comment, marked as answer]
Maybe using custom publishers?

https://goreleaser.com/customization/publishers/
